nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -81,30 +81,30 @@
     protected static final String SENSITIVE_PROPERTY_PREFIX = "SENSITIVE.";
 
     /**
-     * Copied from {@link GenericObjectPoolConfig.DEFAULT_MIN_IDLE} in Commons-DBCP 2.7.0
+     * Copied from {@link GenericObjectPoolConfig#DEFAULT_MIN_IDLE} in Commons-DBCP 2.7.0
      */
-    private static final String DEFAULT_MIN_IDLE = "0";
+    protected static final String DEFAULT_MIN_IDLE = "0";
```

Comment:
It is unused now because `GenericObjectPoolConfig.DEFAULT_MIN_IDLE` is directly referenced in the `PropertyDescriptor`.
Not clear why these constants were copied from `GenericObjectPoolConfig` but `DEFAULT_MAX_IDLE` could also be used directly. Maybe others too.
I'm not in favour of changing them, I'm rather unsure if DEFAULT_MIN_IDLE should be changed at all.

The constants are not referenced any more in the subclass directly, so they could remain `private` I think.